### DISCUSSION TITLE
font size in plot_cpdb3

### DISCUSSION THREAD

**Discussion by Lei-GitHub** [zktuong/ktplots#119] [Q&A]
Hi, May I ask how to change the font size of plot_cpdb3? Thank you very much

**Comment by zktuong** [comment]
Hi @Lei-GitHub,

unfortunately there's no easy way to do it at the moment as the current function does not return the object created by `chordDiagram`

https://github.com/zktuong/ktplots/blob/59caf07290633f481ff0522dbeeedf856661c11c/R/utils.R#L825-L847

An adjustment would be to return the chordDiagram and then i presume you can adjust with `circos.track` and `circos.text`
https://jokergoo.github.io/circlize_book/book/advanced-usage-of-chorddiagram.html#customize-sector-labels

But i don't have a lot of experience with working with the `circlize` package.

**Comment by Lei-GitHub** [comment]
Thank you very much for the quick help. I would appreciate that.
